src/RoutingControllersOptions.ts

```diff
@@ -85,4 +86,28 @@ export interface RoutingControllersOptions {
      */
     currentUserChecker?: CurrentUserChecker;
     
+    /**
+     * Default settings
+     */
+    defaults?: {
+        /**
+         * If set, all null responses will return specified status code by default
+         */
+        nullResultCode?: number;
+
+        /**
+         * If set, all undefined responses will return specified status code by default
+         */
+        undefinedResultCode?: number;
+
+        /**
+         * Default param options
+         */
+        paramOptions?: {
```

Comment:
we already have options for transform/validate. `paramsAlwaysRequired` is less complexier then `paramOptions: { required: true }`. Its pointless to have object for options when there is only one option. If we'll have something in the future we will make deprecations if needed - its easy to do that in here.